README.rst

```diff
@@ -56,11 +53,20 @@ Google Cloud, etc) or any on-premise network storage (via SSH, for example).
 
 The DVC pipelines (computational graph) feature connects code and data together. It is possible to explicitly
 specify all steps required to produce a model: input dependencies including data, commands to run,
-and output information to be saved. See the quick start section below or
+and output information to be saved. See the quick start sections below or
 the `Get Started <https://dvc.org/doc/get-started>`_ tutorial to learn more.
 
-Quick start
-===========
+Quick start (Visual Studio Code)
```

Comment:
I think it's a bit extreme. There should be a single Quick Start, it should be about CLI. We can put it somewhere as a note, or as a separate section in the README (not a second Quick Start though).

Reply:
K